ui/tests/unit/queries.spec.js

```diff
@@ -2,31 +2,43 @@ import { shallowMount } from "@vue/test-utils";
 import Vue from "vue";
 import Vuetify from "vuetify";
 import App from "@/App";
-import { getIndividuals } from "@/apollo/queries";
+//import { getIndividuals } from "@/apollo/queries";
+import * as Queries from "@/apollo/queries";
```

Comment:
Why did you decide to leave this comment?

Reply:
Sorry, I forgot to remove it!